frontend/src/pages/Admin/Subcontractors/index.tsx

```diff
@@ -152,7 +140,7 @@ export const Subcontractors: FC = () => {
                       currentRowData: subcontractorsQuery?.data,
                       apiUrl: `subcontractors/${subcontractorsQuery?.data?.id}`,
                       handleEditMode: handleEditMode,
-                      queryKeys: [`subcontractors - ${currentRowData?.id}`],
+                      queryKeys: [`subcontractors - ${currentRowData?.id}`, `subcontractors`],
```

Comment:
Shawn pointed out on my PR that we only need the first queryKey.
```suggestion
                      queryKeys: [`subcontractors - ${currentRowData?.id}`],
```

Reply:
I believe it's actually necessary to include in order to update the table on the page without having to refresh.